releasenotes/notes/accept-extra-config-files-f0ab26a4c6bc78c0.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    The core Agent now supports multiple configuration files in addition to the main datadog.yaml file. 
+    The -E flag can be used with the ``datadog-agent run`` command to specify multiple configuration paths.
+    Configuration files are merged sequentially. If a key already exists and the foreign value type matches the existing one, the foreign value overrides it.
+    If both the existing value and the new value are nested configurations, they are merged recursively following the same principles.
```

I think we should leave implementations details out, WDYT ?

```suggestion
    The core Agent now supports multiple configuration files in addition to the main ``datadog.yaml`` file. 
    The -E flag can be used with the ``datadog-agent run`` command to specify multiple configuration files to be merged sequentially into the main ``datadog.yaml``.
```